cql3/values.cc

```diff
@@ -55,4 +55,16 @@ raw_value raw_value::make_value(const raw_value_view& view) {
     return make_value(linearized(*view));
 }
 
+raw_value_view raw_value_view::make_temporary(raw_value&& value) {
+    if (!value) {
+        return raw_value_view::make_null();
+    }
+    return raw_value_view(std::move(value).extract_value());
+}
+
+raw_value_view::raw_value_view(bytes&& tmp) {
+    _temporary_storage = make_lw_shared<bytes>(std::move(tmp));
+    _data = fragmented_temporary_buffer::view(bytes_view(*_temporary_storage));
```

Comment:
Does this make `*this` a view to... itself?  If so, then I don't like this design/naming convention.

Reply:
Me neither :) But I also don't like poisoning multiple function interfaces down the call chain just to provide an external structure for temporaries, which are only used in handful of rare cases... I tried both and this one looks like lesser evil. Still, suggestions are welcome.